wicket-extensions/src/main/java/org/apache/wicket/extensions/ajax/AjaxDownloadBehavior.java

```diff
@@ -323,10 +323,30 @@ private String getName()
 	}
 
 	/**
+	 * Controls if anti cache parameter should be added to the URL or not
+	 *
+	 * @return {@code true} to add the anti cache request parameter, {@code false} - otherwise
+	 */
+	protected boolean shouldAddAntiCacheParameter() {
```

Comment:
What is this needed for? The resource controls the caching.

Reply:
During my tests I found the issue: subsequent download was cached. I have implemented this anti-cache, same as in wicket Image.java